docs/onboard.md

```diff
@@ -0,0 +1,86 @@
+# Onboard
+
+## Vendor
+
+### Step 1
+Send in your request for a TechPass account to TechPass Team
```

Comment:
to the product team.  it's our tenant admins who will create these vendor accounts